extensions/persistence/org.eclipse.smarthome.persistence.h2sql/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome H2SQL Persistence Service
+Bundle-SymbolicName: org.eclipse.smarthome.persistence.h2sql
+Bundle-Version: 0.9.0.qualifier
+Bundle-Vendor: openHAB.org
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+DynamicImport-Package: *
+Import-Package: org.apache.commons.io,
+ org.apache.commons.lang,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.library.items,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.persistence,
+ org.eclipse.smarthome.core.types,
+ org.openhab.core.library.types,
```

Comment:
No problem - it's just an error as I originally wrote the H2 persistence forn the OH2 repo.